app/main/controllers/template/RTMediaTemplate.php

```diff
@@ -376,7 +376,8 @@ function save_album_edit() {
 			$submit         = filter_input( INPUT_POST, 'submit', FILTER_SANITIZE_STRING );
 			$_move_selected = filter_input( INPUT_POST, 'move-selected', FILTER_SANITIZE_STRING );
 			$_album         = filter_input( INPUT_POST, 'album', FILTER_SANITIZE_STRING );
-			$_selected      = filter_input( INPUT_POST, 'selected', FILTER_SANITIZE_STRING );
+//			$_selected      = filter_input( INPUT_POST, 'selected', FILTER_SANITIZE_STRING );
+			$_selected = $_POST[ 'selected' ];
```

Comment:
@umeshnevase 

We have to use filter_input for parsing $_POST variable. Please check.